### ISSUE TITLE
Binding news into view models

### ISSUE THREAD

**Issue by altairmizuchi** [badoo/MVICore#154]
Hello, everyone!

I am trying to understand how to bind feature state and news into one ViewModel. Let's assume I have `data class MyViewModel(val users: List<User>, val loadError: String?)` and I have loading error as `News` class with error message text and I have users list as part of  feature `State`
I do not understand how to wire them together. Should I do some magic combining `feature` with its own `feature.news` somehow. Or does it mean I am doing something wrong and News aren't supposed to affect ViewModels.
In other words, what are the best practicies here?

**Comment by arturh** [comment]
Have you considered putting the loading error in the state?
